intranet/static/js/eighth/signupUI.js

```diff
@@ -214,21 +219,22 @@ $(function() {
             listening = false;
         }
     });
-    $("#activity-picker").on("touchstart", function(e) {
+    var $activityPicker = $("#activityPicker");
+    $activityPicker.on("touchstart", function(e) {
         if ($("#activity-picker > .backbtn").hasClass("visible")) {
             e.stopPropagation();
             initX = e.originalEvent.touches[0].clientX;
             initY = e.originalEvent.touches[0].clientY;
             listening = true;
         }
     });
-    $("#activity-picker").on("touchend", function(e) {
+    $activityPicker.on("touchend", function(e) {
         if ($("#activity-picker > .backbtn").hasClass("visible")) {
             e.stopPropagation();
             listening = false;
         }
     });
-    $("#activity-picker").on("touchmove", function(e) {
+    $activityPicker.on("touchmove", function(e) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 6.](https://www.codacy.com/app/tjhsst-ion/ion/file/2688416681/issues/source?bid=3212528&fileBranchId=3299315#l237)